### ISSUE TITLE
"select encoding" didn't work

### ISSUE THREAD

**Issue by amoyonglee** [conwnet/github1s#401]
Lower right button `Select Encoding` didn't work,and document encoding cannot be converted successfully.Display `[object event]`.

environment:
Windows 7, chrome

![8ce41a00976affd407df5dec64a6b4f](https://user-images.githubusercontent.com/49242974/168514434-b3150dde-9b3b-4c2d-a25f-0a8abd95e9c3.png)

**Comment by amoyonglee** [comment]
Thank you for your help,but it still didn't work and throw error `object event`.

Last Friday, someone changed some settings so that this computer could be connected remotely. I don't know whether the error is related to the changes. Do I need to change my local (computer/browser) settings to fix it ?

![image](https://user-images.githubusercontent.com/49242974/168745196-f75c478b-574f-4b51-a2ae-3b383083c6b9.png)

**Comment by conwnet** [comment]
> Thank you for your help,but it still didn't work and throw error object event.

Yes, the changes will take effect in `github1s.com` after they merged into `release` branch

Now it is fixed in `github1s.com`, thank you! Please try again.

**Comment by amoyonglee** [comment]
> > Thank you for your help,but it still didn't work and throw error object event.
> 
> Yes, the changes will take effect in `github1s.com` after they merged into `release` branch
> 
> Now it is fixed in `github1s.com`, thank you! Please try again.

Thank you for your help! And it didn't throw any error when I change document encoding in `github1s.com` today.But it displayed Chinese corrupted code. Here is comparison between github1s.com and vscode.
github1s.com:
![aeb7a7224424300684200cd6bbd7f51](https://user-images.githubusercontent.com/49242974/168945335-49034159-d9a2-42de-ba29-d9f1cf027cef.png)
vs code:
![c40a3b3ba07db19eed41261c8119a82](https://user-images.githubusercontent.com/49242974/168945359-04944b1e-7353-4f8b-b2dd-f77b4b6bcdfc.png)

**Comment by conwnet** [comment]
@amoyonglee Thank you for your information, we will fix this as soon

**Comment by amoyonglee** [comment]
Wow it works successfully !! Thank you very much for your help !!!
